lib/core/notifier.js

```diff
@@ -97,33 +97,28 @@ class NotifierController {
    * @param  {string} message - Additional information
    * @return {Promise}
    */
-  notifyServer (rooms, connectionId, type, message) {
+  async notifyServer (rooms, connectionId, type, message) {
     const channels = [];
 
-    if (rooms.length === 0) {
-      return Bluebird.resolve();
-    }
-
-    for (const room of rooms) {
-      const hotelClerkRoom = this.kuzzle.hotelClerk.rooms.get(room);
+    if (rooms.length > 0) {
```

Comment:
Nit: Why not using a guard instead of indenting the whole method code?

Reply:
Cosmetic change only, I didn't like having that kind of code:

```
if (...) {
  return null;
}

// actual code

if (...) {
  return <some result>;
}

return null;
```

I find this more readable (only works because it's a short function):

```
if (...) {
  // actual code
  if (...) {
    return <some result>;
  }
}

return null;
```

But... that's entirely subjective and I'm not attached to that change. If you prefer I can revert that part to the 1st form.